### PR TITLE
Docs: Added extended Home Assistant example in documentation 

### PR DESCRIPTION
**Pull request by Wetzel402**
## What type of PR is this?

_(REQUIRED)_

- documentation
Added extended Home Assistant example

### CODE REVIEW

File: docs/docs/documentation/community-guide/home-assistant.md [hunk 1]
```diff
@@ -83,3 +83,102 @@ card_mod:
...
+
+#### 2. Create Home Assistant Sensors
+
+Create REST sensors in home assistant
+
+Make sure the url and port (`http://mealie:9000` ) matches your installation's address and _API_ port. Configure the page (`1`), perPage (`5`), and orderDirection (`asc`) as desired. This example will return 1 page with 5 meals per page in ascending order starting today to exclude yesterday's meals. 
+
+```yaml
+rest:
+  - resource_template: http://mealie:9000/api/groups/mealplans?start_date={{ now().strftime('%Y-%m-%d') }}&page=1&perPage=5&orderBy=date&orderDirection=asc
```
Comment (michael-genson) [inline docs/docs/documentation/community-guide/home-assistant.md]:
```suggestion
  - resource_template: http://mealie:9000/api/households/mealplans?start_date={{ now().strftime('%Y-%m-%d') }}&page=1&perPage=5&orderBy=date&orderDirection=asc
```

**Review by michael-genson — CHANGES_REQUESTED** [review]
I can't suggest changes on line 27 and 39, but could you also update those from `.../groups/mealplans...` to `.../households/mealplans...`? The new households PR changes those endpoints

File: docs/docs/documentation/community-guide/home-assistant.md [hunk 2]
```diff
@@ -83,3 +83,102 @@ card_mod:
...
+
+#### 1. Repeat Step 1 Above to get Your API Token
+
+#### 2. Create Home Assistant Sensors
+
+Create REST sensors in home assistant
+
+Make sure the url and port (`http://mealie:9000` ) matches your installation's address and _API_ port. Configure the page (`1`), perPage (`5`), and orderDirection (`asc`) as desired. This example will return 1 page with 5 meals per page in ascending order starting today to exclude yesterday's meals. 
+
+```yaml
```
Comment (joostlek) [inline docs/docs/documentation/community-guide/home-assistant.md:101]: Should we change this to using the integration?

Comment (Wetzel402) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: I think the wording from the previous users is accurate and I'm just reusing it.  We are creating REST sensors for HA.  In HA integration typically means something more advanced though `sensor.rest` is technically an integration too. 

https://www.home-assistant.io/integrations/sensor.rest/
https://www.home-assistant.io/integrations

Comment (joostlek) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: Yes but I'm talking about using the mealie integration to communicate with Mealie.

Comment (Wetzel402) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: Is there a HACS integration?

Comment (michael-genson) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: Mealie is an official integration now: https://www.home-assistant.io/integrations/mealie/

Comment (Wetzel402) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: Considering there is an official integration I may have to rewrite the documentation for this PR.  I will have to explore the integration.  As it stands now my PR doesn't use the integration at all but rather the API and REST sensors.

Comment (joostlek) [reply docs/docs/documentation/community-guide/home-assistant.md:101]: I did add services that you can use to request stuff and then use a template trigger to be able to create template entities from that response. It's a bit more complicated than having ready to go entities, but it is very flexible now

**Pull request closed without merging**